runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -103,6 +103,10 @@
 	public static final int FISHING_CRUSHING_INFERNAL_EELS = 7553;
 	public static final int FISHING_CUTTING_SACRED_EELS = 7151;
 	public static final int FISHING_BAREHAND = 6709;
+	public static final int FISHING_BAREHAND_WINDUP_1 = 6703;
+	public static final int FISHING_BAREHAND_WINDUP_2 = 6704;
+	public static final int FISHING_BAREHAND_CAUGHT_1 = 6710;
+	public static final int FISHING_BAREHAND_CAUGHT_2 = 6711;
```

Comment:
These Barehanded caught animations (and maybe windup ones too) are specific to Tuna, so names are subject to change once we figure out what Shark/Swordfish ones are